doc/guides/collaborator-guide.md

```diff
@@ -143,30 +143,27 @@ the TSC meeting agenda.
 
 ### Waiting for Approvals
 
-Before landing pull requests, allow 48 hours for input from other Collaborators.
-Certain types of pull requests can be fast-tracked and may land after a shorter
-delay. For example:
+Before landing pull requests, allow for input from other Collaborators.
+If you expect that a pull request might raise objections, it is a good idea to
+wait longer before landing. You can also add the `wait-for-feedback` label
+to pull requests; in that case, it is expected that the pull request will not
+be merged until at least 48 hours after it has been opened, in order to allow
+for everybody to comment on it.
 
-* Focused changes that affect only documentation and/or the test suite:
-  * `code-and-learn` tasks often fall into this category.
-  * `good-first-issue` pull requests may also be suitable.
-* Changes that fix regressions:
-  * Regressions that break the workflow (red CI or broken compilation).
-  * Regressions that happen right before a release, or reported soon after.
+For pull requests that have only one approval, it is recommended to keep them
```

Comment:
I would recommend adding “lacking an approval from the relevant team”.

Reply:
I’ll add that but as mentioned in the issue, I think this is something we could improve on a lot through CODEOWNERS.